server/forge/bitbucket/internal/types.go

```diff
@@ -158,7 +158,7 @@ type PullRequestHook struct {
 		Updated time.Time `json:"updated_on"`
 
 		Source struct {
-			Repo   Repo `json:"repsoitory"`
+			Repo   Repo `json:"repository"`
```

Comment:
scary

Reply:
I don't think it was done on purpose. If we would have such a weird  case, I think we would add a comment.